praw/models/reddit/subreddit.py

```diff
@@ -2849,6 +2857,55 @@ def remove(self, redditor: Union[str, "praw.models.Redditor"]):
         self.subreddit._reddit.post(url, data=data)
 
 
+class ModNotes:
+    """TODO
+
+    """
+
+    def __call__(
+        self,
+        redditor: Optional[Union[str, "praw.models.Redditor"]],
+        **generator_kwargs: Any,
+    ) -> Iterator["praw.models.ModNote"]:
+        """TODO
+        """
+        Subreddit._safely_add_arguments(generator_kwargs, "params", subreddit=self.subreddit, user=redditor)
+        url = API_PATH["modnotes"]
+        return ListingGenerator(self.subreddit._reddit, url, **generator_kwargs)
+
+    def __init__(self, subreddit: "praw.models.Subreddit"):
+        """TODO
+        """
+        self.subreddit = subreddit
+
+    def add(
+        self,
+        redditor: Union[str, "praw.models.Redditor"],
+        note: str,
+        label: str = None,
+        reddit_id: str = None,
+        **other_settings: Any
+    ):
+        """TODO
+
+        questions: label has to be one of "BOT_BAN, PERMA_BAN, BAN, ABUSE_WARNING, SPAM_WARNING, SPAM_WATCH, SOLID_CONTRIBUTOR, HELPFUL_USER"
```

Comment:
```suggestion
        questions: label has to be one of ``"BOT_BAN"``, ``"PERMA_BAN"``, ``"BAN"``, ``"ABUSE_WARNING"``, ``"SPAM_WARNING"``, ``"SPAM_WATCH"``, ``"SOLID_CONTRIBUTOR"``, ``"HELPFUL_USER"``.
```
Be sure that the supported labels is formatted like this.